### PR TITLE
SSCS-7578 Changes to support manual upload route

### PR DESCRIPTION
**Pull request by michaellavelle**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-7578

### Change description ###

* Changes to support manual upload route

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java [hunk 1]
```diff
@@ -53,43 +52,48 @@ protected void setTemplateContent(DecisionNoticeOutcomeService outcomeService, P
...
+
+                // If validation has produced no errors, we know that we can get an allowed/refused condition.
+                Optional<EsaAllowedOrRefusedCondition> condition = EsaPointsRegulationsAndSchedule3ActivitiesCondition
+                    .getPassingAllowedOrRefusedCondition(decisionNoticeQuestionService, caseData);
+                if (condition.isPresent()) {
+                    EsaScenario scenario = condition.get().getEsaScenario(caseData);
+                    EsaTemplateContent templateContent = scenario.getContent(payload);
+                    builder.writeFinalDecisionTemplateContent(templateContent);
+                } else {
+                    // Should never happen.
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java:71]: Do we want to log this message to help with debugging?

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java [hunk 2]
```diff
@@ -141,7 +145,8 @@ protected void setDescriptorsAndPoints(WriteFinalDecisionTemplateBodyBuilder bui
             .stream().map(q ->
                 buildDescriptorFromActivityQuestion(activityQuestionlookup.getByKey(q))).collect(Collectors.toList());
 
-        descriptors.sort(new DescriptorLexicographicalComparator());
+        // FIXME Extract sort order from question text
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java:148]: Will this be fixed as part of separate PR?

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java (hunk 1 above)
Comment (michaellavelle) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java:71]: Good idea - now logging

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java (hunk 2 above)
Comment (michaellavelle) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java:149]: Yes, as discussed
